src/clr-base/components/clr-wc-element/clr-wc.element.ts

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
So we are going with `clr-wc.*.[ts|scss|spec.ts]`?

Reply:
We dont need the prefix clr-wc I think that is just what we named the demo component. We do need `.element` though so I can glob match on the transpiled code to list which files have side effects. See the placeholder button and icon component for the convention.